src/components/dateTimePicker.js

```diff
@@ -41,6 +44,52 @@
     const [selectedDate, setSelectedDate] = useState(null);
     const helper = useText(helperText);
     const placeholderText = useText(placeholder);
+    const parsedMinDate =
```

Comment:
Using a `let` instead of a `const` might be better here. This way you can avoid disabling the linter. Something like this would do:

```
let parsedMinDate;
let parsedMaxDate;
if (useMinMaxDate === 'date') {
  parsedMinDate = new Date(useText(options.minDate));
  parsedMaxDate = new Date(useText(options.maxDate));
} else if (useMinMaxDate === 'days') {
  parsedMinDate = useText(minDaysBefore);
  parsedMaxDate = useText(maxDaysAhead);
}
```
There is nothing wrong with using a `let`. And it will make this code easier and saves some lines.